Flickable: reject Wheel events in the orthogonal direction (#8816)
slint-ui/slint: docs/astro/src/content/docs/reference/gestures/flickable.mdx

   a `TouchArea`, then `Flickable` will flick immediately on pointer move events when the euclidean distance
   to the coordinates of the press event exceeds 8 logical pixels.

## Wheel/Scroll Event Interaction

> The Flickable also supports scrolling with the mouse wheel and touchpad scroll gestures.

Reviewer: ```suggestion
The `Flickable` also supports scrolling with the mouse wheel and touchpad scroll gestures.
```

It will scroll regardless of the `interactive` property.

> If the event is in a direction that for which the Flickable can scroll, the event will be intercepted.

Reviewer: ```suggestion
If the `Flickable` can scroll in the event's direction, the event will be intercepted.
```
